README.md

```diff
@@ -3,21 +3,50 @@
 [![Node.js CI](https://github.com/libsql/libsql-client-ts/actions/workflows/ci.yaml/badge.svg)](https://github.com/libsql/libsql-client-ts/actions/workflows/ci.yaml)
 [![License](https://img.shields.io/badge/license-MIT-blue)](https://github.com/libsql/libsql-client-ts/blob/main/LICENSE)
 
-This is the source repository of the JavaScript & TypeScript SDK for libSQL. You can either connect to a local SQLite/libSQL database (embedded in the client) or to a remote libSQL server.
+This is the source repository of the JavaScript & TypeScript SDK for libSQL. You can use it to interact with the following types of databases:
+
+- Local SQLite database files
+- [libSQL sqld] instances (including [Turso])
 
 ## Installation
 
-```
+```shell
 npm install @libsql/client
 ```
 
-## Getting Started
+## Create a database client object
+
+### Importing
 
-Connecting to a local SQLite/libSQL database:
+There are multiple ways to import the module. For Node.js and other environments where want to use a local SQLite [file URL](#local-sqlite-file-urls), as well as network access to `sqld`:
```

Comment:
The "want" in the second sentence is missing a subject; perhaps this should be "where you want to use"?

Reply:
Also, Node.js is the *only* environment where local SQLite will work, there are no "other environments" distinct from Node that we can support.

Reply:
Will update in another PR.